tests/test_sse.py

```diff
@@ -6,8 +6,8 @@
 from aiohttp_sse import EventSourceResponse
 
 
-@pytest.mark.run_loop
-def test_func(loop, unused_port):
+@pytest.mark.asyncio
```

Comment:
Should we use `pytest.mark.asyncio(forbid_global_loop=True)` instead? Previous implementation had `asyncio.set_event_loop(None)` so we can get handy error if loop is not passed explicitly somewhere.